unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/HUDController.cs

```diff
@@ -269,7 +269,7 @@ private void AllUIHiddenOnOnChange(bool current, bool previous)
 
                     if (friendsHud != null)
                     {
-                        friendsHud.Initialize(FriendsController.i, UserProfile.GetOwnUserProfile(), socialAnalytics);
+                        friendsHud.Initialize(FriendsHUDComponentView.Create());
```

Comment:
We usually solve the mocking issue here instead of using DI, just having a virtual method `CreateView` in the Controller that can be changed using `Substitute.ForParts<>()`.

Not saying I dont like this approach, but we should keep a common approach.

Reply:
Null checking is not needed (or is not a need raised from having a CreateView method) you could pass a null view through dependency injection as well.

I agree I rather have DI though.